app/views/warehouse_reports/outcomes/rrh_report/README.md

```diff
@@ -110,8 +110,8 @@ A successful exit to permanent housing is defined as an exit to permanent housin
 
 ## Returns to Homelessness from Permanent Housing
 
-The number of unique clients that with a subsequent enrollment after successfully exiting Rapid Rehousing to a permanent housing destination.
+The number of unique clients who had moved into, and then exited permanent housing to a permanent destination, and subsequently enrolled in a homeless project.
```

Comment:
The two kinds of permanent read awkwardly, but I don't have a better suggestion.